### DISCUSSION TITLE
Password encryption

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5505] [Q&A] [answered]
I create blueprint and add in it some fields one of them is PASSWORD, when submit the password not encrypted into database. How to make this please ?

**Comment by jackmcdade** [comment, marked as answer]
Can you explain a bit more? Is this a user blueprint or a collection blueprint? What fieldtype is your password field?

**Reply by HazemMohamed252** [reply]
This is new blueprint call: Client, it is under COLLECTIONS section, fieldtype for password is text and i made it html password type

**Reply by jackmcdade** [reply]
Ah okay – that field doesn't do any server-side logic, it just uses the password input HTML type. If we encrypted it there would be no way for you to check against it.

If you need users that can log into your application or statamic site, we highly recommend using actual [Users](https://statamic.dev/users) because all that functionality is already built in. Alternatively, you could [build your own custom fieldtype](https://statamic.dev/extending/fieldtypes) that handled encrypting, etc.

**Reply by HazemMohamed252** [reply]
What i need is to have module like USERS call: Client and the super admin can CRUD a client from the CP. How can i do that or how to use users to make super admin create client for my site. I wish you can understand me clairly. Thanks

**Reply by jackmcdade** [reply]
This is how you create users in Statamic: https://statamic.dev/users#creating-users

**Reply by HazemMohamed252** [reply]
I understand that and i know it is clear but this section for CP access but what if i need users for front-end module, the admin can create one as it is registered from the front-end

**Reply by jackmcdade** [reply]
Then you would create a new Role that wouldn't have CP Access and create users with that role using the same same features.

**Reply by HazemMohamed252** [reply]
Thanks for your support
